### DISCUSSION TITLE
Only greeting the new contributors

### DISCUSSION THREAD

**Discussion by ghost** [MakeContributions/DSA#241] [General]
I think we should greet new contributors only since after a few contribution we get the idea about the contribution guidelines and also the greeting message in each issue and PR also clutters them a bit.

We can use the [welcome bot](https://github.com/apps/welcome) for this, we only need to specify the greeting messages

**Comment by TawfikYasser** [comment]
@atin  Hi,
This is a good idea

If you can do that raise a PR

**Reply by ming-tsai** [reply]
May be for first time contributor

**Reply by ghost** [reply]
@ming-tsai we have to install the welcome bot in this repo, i think only you can do that, i've sent you a request for that.

**Reply by ghost** [reply]
@ming-tsai Have you accepted the request? You can also add it your self from [here](https://github.com/apps/welcome)
